avm/server/src/runner.rs

```diff
@@ -110,6 +110,11 @@ impl AVMRunner {
             max_memory_size: stats[0].max_memory_size,
         }
     }
+
+    #[inline]
+    pub fn set_peer_id(&mut self, current_peer_id: impl Into<String>) {
```

Comment:
This method is used in tests only.  Should I make it really public and write tests for it, or make it `hidden`?

Reply:
Nope, it doesn't needed, just leave as is.